assets/sass/components/settings/_googlesitekit-settings-user-input.scss

```diff
@@ -0,0 +1,32 @@
+/**
+ * Settings user input styles.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-settings-user-input {
+		padding-bottom: 0;
+
+		.googlesitekit-user-input__preview {
+			padding: $grid-gap-desktop;
+
+			@media (max-width: $bp-mobileOnly) {
+				padding: $grid-gap-phone;
+			}
```

Comment:
This should be the other way around – mobile styles first and then media query with `min-width` for larger viewports 🙂  Almost all of our styles are this way.